modules/twinkleblock.js

```diff
@@ -163,6 +170,33 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 
 	partial.prop('disabled', !blockBox && !templateBox);
 
+	// Add current block parameters as default preset
+	if (Twinkle.block.currentBlockInfo) {
```

Comment:
This could be changed if we only wanted the original settings to be available on the same type (partial or not) of block.  I don't think that's better, but something like `((partialBox && Twinkle.block.currentBlockInfo.partial === '') || (!partialBox && Twinkle.block.currentBlockInfo.partial !== ''))` would do it.